modules/containers-signature-verify-enable.adoc

```diff
@@ -94,3 +95,91 @@ $ butane 51-master-rh-registry-trust.bu -o 51-master-rh-registry-trust.yaml
 ----
 $ oc apply -f 51-master-rh-registry-trust.yaml
 ----
+
+. Check that the worker machine config pool has rolled out with the new machine config:
+
+.. Check that the new machine config was created:
++
+[source,terminal]
+----
+$ oc get mc
+----
++
+.Sample output
+[source,terminal] 
+----
+NAME                                               GENERATEDBYCONTROLLER                      IGNITIONVERSION   AGE
+00-master                                          a2178ad522c49ee330b0033bb5cb5ea132060b0a   3.2.0             25m
+00-worker                                          a2178ad522c49ee330b0033bb5cb5ea132060b0a   3.2.0             25m
+01-master-container-runtime                        a2178ad522c49ee330b0033bb5cb5ea132060b0a   3.2.0             25m
+01-master-kubelet                                  a2178ad522c49ee330b0033bb5cb5ea132060b0a   3.2.0             25m
+01-worker-container-runtime                        a2178ad522c49ee330b0033bb5cb5ea132060b0a   3.2.0             25m
+01-worker-kubelet                                  a2178ad522c49ee330b0033bb5cb5ea132060b0a   3.2.0             25m
+51-master-rh-registry-trust                                                                   3.2.0             13s <1>
+51-worker-rh-registry-trust                                                                   3.2.0             53s
+99-master-generated-crio-seccomp-use-default                                                  3.2.0             25m
+99-master-generated-registries                     a2178ad522c49ee330b0033bb5cb5ea132060b0a   3.2.0             25m
+99-master-ssh                                                                                 3.2.0             28m
+99-worker-generated-crio-seccomp-use-default                                                  3.2.0             25m
+99-worker-generated-registries                     a2178ad522c49ee330b0033bb5cb5ea132060b0a   3.2.0             25m
+99-worker-ssh                                                                                 3.2.0             28m
+rendered-master-af1e7ff78da0a9c851bab4be2777773b   a2178ad522c49ee330b0033bb5cb5ea132060b0a   3.2.0             8s  <2>
+rendered-master-cd51fd0c47e91812bfef2765c52ec7e6   a2178ad522c49ee330b0033bb5cb5ea132060b0a   3.2.0             24m
+rendered-worker-2b52f75684fbc711bd1652dd86fd0b82   a2178ad522c49ee330b0033bb5cb5ea132060b0a   3.2.0             24m
+rendered-worker-be3b3bce4f4aa52a62902304bac9da3c   a2178ad522c49ee330b0033bb5cb5ea132060b0a   3.2.0             48s
+----
+<1> New machine config
+<2> New rendered machine config 
+
+.. Check that the new machine config pool was created:
```

Comment:
modified to: Check that the machine config pool worker is updating with the new machine config. When 'UPDATING' field become 'False', it mean the machine config pool worker has rolled out to the new machine config. You can do the next step.

(Because the mcp master and worker are created by default, it's not proper to say "the new machine config pool was created")

Reply:
@lyman9966 I made the changes as suggested. Thank you.